src/pages/components/UserInformation/UserInfo.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Box, Text, useBreakpointValue } from '@chakra-ui/react';
+
+const UserInfo: React.FC = function UserInfo() {
```

Comment:
instead of hardcoding these values, it's probably expected that this `UserInfo` component will take these values (i.e. name, email, date joined) in as props

update this component so that these values are props instead of hardcoded

Reply:
Sure. However, I'll need to integrate the auth endpoint to do this. Still on designs now, I'll update ASAP.